wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/WireMockApp.java

```diff
@@ -27,4 +28,8 @@ public void stubFor(MappingBuilder mappingBuilder) {
   public StubRequestHandler buildStubRequestHandler() {
```

Comment:
obvious methods; could be removed?

Reply:
not really, as it delegates to original wm class